### DISCUSSION TITLE
505 after adding a user and password in config.php

### DISCUSSION THREAD

**Discussion by steph-dmts** [YOURLS/YOURLS#3345] [Q&A] [answered]
Hi,

I had forgotten my Admin password so I've added a new user and password in the config.php file.
After that, every request I make to the /admin/index.php URL and every request I make on actual short links results in a 505 error.
Any help would be greatly appreciated.

Thank you

**Comment by steph-dmts** [comment, marked as answer]
Silly mistake, I actually forgot to put a comma after the last key-value pair before closing the array of passwords like so:
```php
$yourls_user_passwords = [
        'user' => 'password',
        'user2' => 'password'  // <- there should be a comma here as well
];
```

**Reply by dgw** [reply]
This makes no sense? PHP doesn't care if an array has a trailing comma. HTTP 505 ("HTTP Version Not Supported") makes no sense either.

But what's important is, it sounds like you fixed the problem, somehow. 😸

**Comment by steph-dmts** [comment]
I agree that the error message makes little sense it should be 500 not 505.
The issue was that the config file couldn't be compiled and, even though apparently PHP doesn't care fore trailing commas, that fixed it.
